code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -198,6 +198,9 @@
 		message_admins(span_danger("ERROR: Non-admin [key_name(usr)] attempted to execute a SDQL query!"))
 		log_admin("Non-admin [key_name(usr)] attempted to execute a SDQL query!")
 		return FALSE
+	var/prompt = tgui_alert(usr, "Run SDQL2 Query?", "SDQL2", list("Yes", "Cancel"))
```

Comment:
Use alert so SDQL is available if tgui isn’t

Reply:
TGUI should always be available but our TGUI is probably outdated so probably doesn't. I'll change it but it would be better to have tgui more functional.